src/main/webapp/js/user-page-loader.js

```diff
@@ -29,18 +29,15 @@ function setPageTitle() {
   document.title = parameterUsername + ' - User Page';
 }
 
-/**
- * Shows the message form if the user is logged in and viewing their own page.
- */
-function showMessageFormIfViewingSelf() {
+function showMessageFormIfLoggedIn() {
```

Comment:
Please leave the comment in, but remove the part where it says "and viewing their own page". Thanks!